Python/crossinterp.c

```diff
@@ -1485,6 +1491,10 @@ _PyXI_excinfo_Apply(_PyXI_excinfo *info, PyObject *exctype)
         if (tbexc == NULL) {
             PyErr_Clear();
         }
+        else {
+            PyErr_SetObject(exctype, tbexc);
+            return;
```

Comment:
`tbexc` seems to be leaking.

Reply:
fixed

Thanks for tracking this down!  I don't know how you find these things so quickly. 😄